### ISSUE TITLE
Mapbox zoom LatLngBound sometimes wrong

### ISSUE THREAD

**Issue by thanhnguyenduc157** [mapbox/mapbox-navigation-android#880]
When I use  navigationView for my project, i have a function to zoom map in the center of start point and end point:

navigationView..getMapboxMap().easeCamera(CameraUpdateFactory.newLatLngBounds(latLngBounds, 10));

but the strange thing happen: some time it zoom right, some time it zoom wrong (not move to center of LatLngBound).

**Comment by LukasPaczos** [comment]
Thanks for reporting the issue @thanhnguyenduc157.
This is resolved with https://github.com/mapbox/mapbox-gl-native/pull/11410 that is a part of https://github.com/mapbox/mapbox-navigation-android/pull/697.

**Comment by thanhnguyenduc157** [comment]
so i need upgrade Mapbox Android SDK, right @LukasPaczos ?

**Comment by LukasPaczos** [comment]
Unfortunately, you can't yet because the navigation `v0.12.0` and Maps SDK `v6.x` are incompatible. You will have to wait until #697 is merged and released.

**Comment by danesfeder** [comment]
Thanks @LukasPaczos, @thanhnguyenduc157 let's close this issue when #697 is merged.  Thank you for bringing this to us!

**Comment by Guardiola31337** [comment]
@thanhnguyenduc157 hey 👋 friendly reminder that this should be fixed now (#697 landed last week 🎉) and it'll be available in the next release. If you'd like, you could double check this using `0.13.0-SNAPSHOT`.

Following you can find installation instructions for `SNAPSHOT`s:

```java
repositories {
    jcenter()
    maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
    // ...
}

dependencies {
    // ...
    implementation 'com.mapbox.mapboxsdk:mapbox-android-navigation:0.13.0-SNAPSHOT'
    implementation 'com.mapbox.mapboxsdk:mapbox-android-navigation-ui:0.13.0-SNAPSHOT'
    // ...
}
```

Thanks again!
